src/cpp/shared/scripting/callback.cpp

```diff
@@ -17,9 +18,4 @@ void ironfist_lua_push(std::string arg) {
 }
 
 void ironfist_lua_pushmulti() {
-};
-
-void ScriptCallback(const char * funcName) {
```

Comment:
Have you tested using ScriptCallback with only funcName (without args) after removing this? I vaguely recall something wasn't working for me until I added this one

Reply:
Yes. I remember that discussion and was surprised; I don't remember what the story was behind that. But, the code wouldn't compile if this didn't work.  OnMapStart was called successfully, and does not take any parameters.